src/tools/kata-ctl/src/check.rs

```diff
@@ -197,6 +212,81 @@ pub fn check_official_releases() -> Result<()> {
     Ok(())
 }
 
+#[cfg(any(target_arch = "x86_64"))]
+pub fn check_kernel_module_loaded(module: &str, parameter: &str) -> Result<String, String> {
```

Comment:
Can you add a comment here listing out the steps you are using for checking the module?

Reply:
comments added, thanks @amshinde